Automattic/wp-calypso
# Google Photos: Update SVG logo

Related to #

## Proposed Changes

* Update Google Photos SVG logo

## Why are these changes being made?


* Remove empty space around the logo

## Testing Instructions



* Make sure that you don't have a Google Photos connection
* Go to the `Media` page
* Select "Google Photos" option
* Check if there is a better spacing around the Google Photos logo

| Before | After |
|--------|--------|
| <img width="1311" alt="Screenshot 2024-11-29 at 00 34 12" src="https://github.com/user-attachments/assets/94fddd55-9022-495f-94cd-c5afb765fea3"> | <img width="1312" alt="Screenshot 2024-11-29 at 00 25 09" src="https://github.com/user-attachments/assets/cf163b91-75b6-46b2-a33e-c3c3e8d36a29"> | 


## Pre-merge Checklist
